examples/docker/pyproject.toml

```diff
@@ -14,3 +14,42 @@ classifiers = [
   "Programming Language :: Python :: 3.12",
 ]
 requires-python = ">=3.12"
+
+[tool.pixi.project]
+channels = ["conda-forge"]
+platforms = ["linux-64", "linux-aarch64", "osx-arm64"]
+
+[tool.pixi.dependencies]
+python = ">=3.12"
+flask = ">=3.0.2,<3.1"
+gunicorn = ">=21.2.0,<21.3"
+
+[tool.pixi.feature.dev.pypi-dependencies]
+docker-project = { path = ".", editable = true }
+
+[tool.pixi.host-dependencies]
+pip = "*"
+
+[tool.pixi.feature.test.dependencies]
+pytest = "*"
+
+[tool.pixi.feature.build.dependencies]
+build = "*"
+hatchling = "*"
+
+[tool.pixi.tasks]
+dev = "flask --app docker_project:app run"
+start = "gunicorn -w 4 docker_project:app --bind :8000"
+
+[tool.pixi.feature.test.tasks]
+test = "pytest"
+
+[tool.pixi.feature.build.tasks]
+build-wheel = "python -m build --no-isolation --wheel ."
+
+[tool.pixi.feature.prod.tasks]
+postinstall-production = "pip install --no-deps --disable-pip-version-check dist/docker_project-*.whl"
+
+[tool.pixi.environments]
+default = ["test", "build", "dev"]
+prod = ["prod"]
```

Comment:
Both envs need to be in the same solve-group, otherwise this example is a bit pointless.

Reply:
Yes. This posed a problem, so it was good we tested this. Made an issue for it here: #1123